src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -288,6 +288,8 @@ def get(self, request: Request, project: Project) -> Response:
                             )
                     except Exception:
                         logger.exception("Something unexpected happen. Continuing execution.")
+                    # We don't expect coverage data if the integration does not exist (404)
+                    scope.set_tag("codecov.enabled", should_get_codecov_data)
```

Comment:
Since it is within the if block we could just set it to True.
<img width="166" alt="image" src="https://user-images.githubusercontent.com/44410/214316046-15d97863-db5d-49c4-9f18-6f6d57adb0fd.png">
```suggestion
                    scope.set_tag("codecov.enabled", True)
```